src/UsgsAstroFrameSensorModel.cpp

```diff
@@ -15,14 +15,15 @@ using namespace std;
 // Declaration of static variables
 const std::string UsgsAstroFrameSensorModel::_SENSOR_MODEL_NAME
                                       = "USGS_ASTRO_FRAME_SENSOR_MODEL";
-const int UsgsAstroFrameSensorModel::m_numParameters = 6;
+const int UsgsAstroFrameSensorModel::m_numParameters = 7;
 const std::string UsgsAstroFrameSensorModel::m_parameterName[] = {
   "X Sensor Position (m)",  // 0
   "Y Sensor Position (m)",  // 1
   "Z Sensor Position (m)",  // 2
-  "Omega (radians)",        // 3
-  "Phi (radians)",          // 4
-  "Kappa (radians)"         // 5
+  "w",        // 3
+  "x",        // 4
+  "y"         // 5
+  "z"         // 5
 };
```

Comment:
These should be v1, v2, v3 for x, y, z, respectively.

Reply:
This also means that the pulling of these in the calculation of the rotation matrices should probably use the proper variable names. Instead of the lazy x,y,z